<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>108</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">HistorgamsReader</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="World"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Switch" name="switch_historgamming">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>5</y>
      <width>18</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_historgamming">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>15</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogramming</string>
    </property>
   </widget>
   <widget class="Switch" name="switch_filter">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>35</y>
      <width>18</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>45</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>BC filter</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="prefix">
     <string>0x</string>
    </property>
    <property name="maximum">
     <number>3563</number>
    </property>
    <property name="value">
     <number>3563</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_read">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>101</width>
      <height>65</height>
     </rect>
    </property>
    <property name="title">
     <string>Link status</string>
    </property>
    <widget class="QLabel" name="label_icon_clock">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>40</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../!images/img.qrc">:/1G.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_trigger">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>22</y>
       <width>70</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>trigger sync:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_icon_trigger">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>17</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../!images/img.qrc">:/1G.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_clock">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>45</y>
       <width>70</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>clock sync:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>27</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">A0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">C0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">C1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">C2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_PM">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>30</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PM:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Switch</class>
   <extends>QWidget</extends>
   <header>switch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../!images/img.qrc"/>
 </resources>
 <connections/>
</ui>
